fitbenchmarking/fitbenchmarking
# Refactoring html pages to use a template

#### Description of Work

Fixes #309 

Removes hard coded rst tables and then uses a template for the supporting HTML tables

#### Testing Instructions

1. Check supporting links work and pages are consistent with those before.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
